app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/TabInCollectionViewHolder.kt

```diff
@@ -4,55 +4,53 @@
 
 package org.mozilla.fenix.home.sessioncontrol.viewholders
 
-import android.graphics.Outline
-import android.view.View
-import android.view.ViewOutlineProvider
+import android.os.Build
+import android.os.Build.VERSION.SDK_INT
 import androidx.appcompat.content.res.AppCompatResources
-import kotlinx.android.synthetic.main.list_element.*
+import mozilla.components.browser.icons.BrowserIcons
 import mozilla.components.feature.tab.collections.TabCollection
+import mozilla.components.lib.publicsuffixlist.PublicSuffixList
 import mozilla.components.support.ktx.android.content.getColorFromAttr
-import mozilla.components.support.ktx.android.util.dpToFloat
+import mozilla.components.support.ktx.android.content.res.resolveAttribute
+import mozilla.components.ui.widgets.WidgetSiteItemView
 import org.mozilla.fenix.R
-import org.mozilla.fenix.utils.view.ViewHolder
 import org.mozilla.fenix.ext.components
-import org.mozilla.fenix.ext.increaseTapArea
 import org.mozilla.fenix.ext.loadIntoView
 import org.mozilla.fenix.ext.toShortUrl
 import org.mozilla.fenix.home.sessioncontrol.CollectionInteractor
+import org.mozilla.fenix.utils.view.ViewHolder
 import mozilla.components.feature.tab.collections.Tab as ComponentTab
 
 class TabInCollectionViewHolder(
-    view: View,
+    private val view: WidgetSiteItemView,
     val interactor: CollectionInteractor,
-    private val differentLastItem: Boolean = false
+    private val icons: BrowserIcons = view.context.components.core.icons,
+    private val publicSuffixList: PublicSuffixList = view.context.components.publicSuffixList
 ) : ViewHolder(view) {
 
     lateinit var collection: TabCollection
         private set
     lateinit var tab: ComponentTab
         private set
     var isLastItem = false
+        private set
 
     init {
-        list_item_favicon.clipToOutline = true
-        list_item_favicon.outlineProvider = object : ViewOutlineProvider() {
-            override fun getOutline(view: View, outline: Outline?) {
-                outline?.setRoundRect(
-                    0,
-                    0,
-                    view.width,
-                    view.height,
-                    FAV_ICON_BORDER_RADIUS_IN_DP.dpToFloat(view.context.resources.displayMetrics)
-                )
-            }
+        if (SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
How are we handling this for v21-22?

Reply:
We don't show a foreground, just like other views